### PR TITLE
feat: separate quotes locale messages

### PR DESCRIPTION
**Pull request by NaMax66**
### Jira-link:
https://virtocommerce.atlassian.net/browse/VCST-1794
### Artifact URL:
https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-2.6.0-pr-1313-e46e-e46e412a.zip

### CODE REVIEW

**Review by asvishnyakov — COMMENTED** [review]
Why introduce breaking changes by changing localization paths (`pages.account.quote*` -> `quotes.*`)?

And vue i18n already have ability to merge multiple localization files: https://kazupon.github.io/vue-i18n/api/#methods we don't need to do it manually

File: client-app/core/composables/useLanguages.ts [hunk 1]
```diff
@@ -98,6 +100,39 @@ function detectLocale(locales: unknown[]): string {
...
+      import(`../../modules/${moduleName}/locales/${locale}.json`).catch((error) => {
+        Logger.error(`Locale: ${locale} for the module ${moduleName} not found`, error);
+
+        return {};
+      }),
+    ]);
+
+    const existingMessages = i18n.global.getLocaleMessage(locale as string);
+
+    i18n.global.setLocaleMessage(locale as string, {
```
Comment (ivan-kalachikov) [inline client-app/core/composables/useLanguages.ts:125]: why don't use `mergeLocaleMessage` ?

**Comment by NaMax66** [comment]
> Why introduce breaking changes by changing localization paths (`pages.account.quote*` -> `quotes.*`)?
> 
> And vue i18n already have ability to merge multiple localization files: https://kazupon.github.io/vue-i18n/api/#methods we don't need to do it manually

This emphasizes the independence of the Quotes module.

File: client-app/core/composables/useLanguages.ts (hunk 1 above)
Comment (ivan-kalachikov) [reply client-app/core/composables/useLanguages.ts]: weird, but okay, I think it's not critical

**Review by ivan-kalachikov — APPROVED** [review]

**Pull request merged**